packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -102,13 +104,25 @@ export function ScreenshotEditorFactory({
         }
       }
 
-      const cropButton = cropContainerRef.current;
-      if (cropButton) {
-        cropButton.style.width = `${imageDimensions.width}px`;
-        cropButton.style.height = `${imageDimensions.height}px`;
+      const cropContainer = cropContainerRef.current;
+      if (cropContainer) {
+        cropContainer.style.width = `${imageDimensions.width}px`;
+        cropContainer.style.height = `${imageDimensions.height}px`;
       }
 
       setCroppingRect({ startX: 0, startY: 0, endX: imageDimensions.width, endY: imageDimensions.height });
+
+      const annotater = annotatingRef.current;
+      if (annotater) {
+        annotater.width = imageDimensions.width * DPI;
+        annotater.height = imageDimensions.height * DPI;
+        annotater.style.width = `${imageDimensions.width}px`;
+        annotater.style.height = `${imageDimensions.height}px`;
+        const ctx = annotater.getContext('2d');
+        if (ctx) {
+          ctx.scale(DPI, DPI);
+        }
+      }
```

Comment:
since this is mostly duplicate code, it might be worth refactoring this into a function to save some bytes.